### PR TITLE
pythonPackages.pykka: 2.0.1 -> 2.0.3

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pykka/default.nix [hunk 1]
```diff
@@ -1,27 +1,20 @@
...
+  checkInputs = [ pytestCheckHook pytest-mock ];
 
-  meta = with stdenv.lib; {
-    homepage = "http://www.pykka.org";
+  meta = with lib; {
+    homepage = "https://www.pykka.org/";
     description = "A Python implementation of the actor model";
+    changelog = "https://github.com/jodal/pykka/blob/v${version}/docs/changes.rst";
     license = licenses.asl20;
-    maintainers = [];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pykka/default.nix:24]: We should probably add one maintainer here.

File: pkgs/development/python-modules/pykka/default.nix [hunk 2]
```diff
@@ -1,27 +1,20 @@
-{ stdenv
-, buildPythonPackage
-, fetchFromGitHub
-}:
+{ lib, buildPythonPackage, fetchPypi, pytestCheckHook, pytest-mock }:
 
 buildPythonPackage rec {
-  pname = "pykka";
-  version = "2.0.1";
+  pname = "Pykka";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pykka/default.nix:4]: Does this require a rename in all-packages?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107629` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>14 packages built:</summary>
  <ul>
    <li>mopidy-iris</li>
    <li>mopidy-local</li>
    <li>mopidy-mopify</li>
    <li>mopidy-mpd</li>
    <li>mopidy-mpris</li>
    <li>mopidy-musicbox-webclient</li>
    <li>mopidy-somafm</li>
    <li>mopidy-soundcloud</li>
    <li>mopidy-spotify-tunigo</li>
    <li>mopidy-tunein</li>
    <li>mopidy-youtube</li>
    <li>python37Packages.pykka</li>
    <li>python38Packages.pykka</li>
    <li>python39Packages.pykka</li>
  </ul>
</details>

File: pkgs/development/python-modules/pykka/default.nix (hunk 2 above)
Comment (marsam) [reply pkgs/development/python-modules/pykka/default.nix:4]: I think it's not necessary since it would _just_ change the derivation name

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107629` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>mopidy-musicbox-webclient</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>mopidy-spotify-tunigo</li>
  </ul>
</details>
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li>mopidy-iris</li>
    <li>mopidy-local</li>
    <li>mopidy-mopify</li>
    <li>mopidy-mpd</li>
    <li>mopidy-mpris</li>
    <li>mopidy-somafm</li>
    <li>mopidy-soundcloud</li>
    <li>mopidy-tunein</li>
    <li>mopidy-youtube</li>
    <li>python37Packages.pykka</li>
    <li>python38Packages.pykka</li>
    <li>python39Packages.pykka</li>
  </ul>
</details>

mopidy-spotify-tunigo:

```
  trying https://developer.spotify.com/download/libspotify/libspotify-12.1.51-Darwin-universal.zip                                                                                              % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current                                                                                                                                              Dload  Upload   Total   Spent    Left  Speed                                                                                                                 0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0                                                                                                              curl: (22) The requested URL returned error: 404                                                                                                                                            error: cannot download libspotify-12.1.51-Darwin-universal.zip from any mirror 
```

File: pkgs/development/python-modules/pykka/default.nix (hunk 1 above)
Comment (jonringer) [reply pkgs/development/python-modules/pykka/default.nix:24]: or remove the package

Comment (marsam) [reply pkgs/development/python-modules/pykka/default.nix:24]: I added myself as maintainer :)

**Comment by marsam** [comment]
@GrahamcOfBorg build mopidy

File: pkgs/development/python-modules/pykka/default.nix (hunk 2 above)
Comment (jonringer) [reply pkgs/development/python-modules/pykka/default.nix:4]: it's not needed. Attrs shoudl now follow PEP 0503

According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107629` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>14 packages built:</summary>
  <ul>
    <li>mopidy-iris</li>
    <li>mopidy-local</li>
    <li>mopidy-mopify</li>
    <li>mopidy-mpd</li>
    <li>mopidy-mpris</li>
    <li>mopidy-musicbox-webclient</li>
    <li>mopidy-somafm</li>
    <li>mopidy-soundcloud</li>
    <li>mopidy-spotify-tunigo</li>
    <li>mopidy-tunein</li>
    <li>mopidy-youtube</li>
    <li>python37Packages.pykka</li>
    <li>python38Packages.pykka</li>
    <li>python39Packages.pykka</li>
  </ul>
</details>

**Pull request merged**